airflow-core/src/airflow/dag_processing/manager.py

```diff
@@ -884,32 +884,52 @@ def client(self) -> Client:
     def _create_process(self, dag_file: DagFileInfo) -> DagFileProcessorProcess:
         id = uuid7()
 
-        callback_to_execute_for_file = self._callback_to_execute.pop(dag_file, [])
+        callbacks = self._callback_to_execute.pop(dag_file, [])
         logger, logger_filehandle = self._get_logger_for_dag_file(dag_file)
 
-        return DagFileProcessorProcess.start(
-            id=id,
-            path=dag_file.absolute_path,
-            bundle_path=cast("Path", dag_file.bundle_path),
-            callbacks=callback_to_execute_for_file,
-            selector=self.selector,
-            logger=logger,
-            logger_filehandle=logger_filehandle,
-            client=self.client,
-        )
+        try:
+            # Attempt to start the subprocess for parsing
+            processor = DagFileProcessorProcess.start(
+                id=proc_id,
```

Comment:
```suggestion
                id=id,
```
otherwise it's undefined

Reply:
@zachliu thank you!